docs/security/sasl-based-ldap-variables.md

```diff
@@ -0,0 +1,161 @@
+# SASL-based LDAP variables
+
+## Variables installed
+
+The installation adds the following variables:
+
+| Variable name | Description |
+|---|---|
+| authentication_ldap_sasl_bind_base_dn | Base distinguished name |
```

Comment:
It seems that some variables have links, others don't have one. Is this intentional?

Reply:
Yes. I can only link to the variables with details. If you want to provide the details for the rest of the variables I'll add them.